docs/layer-contents.md

```diff
@@ -23,12 +23,9 @@ For other uses it's recommended to base images on `core-image-minimal` or
 `core-image-base` as appropriate. The old image names (`rpi-hwup-image` and
 `rpi-basic-image`) are deprecated.
 
-## WiFi firmware blobs
-
-Be aware that the WiFi firmwares for the supported boards are not provided by
-`linux-firmware` but by a custom recipe which only packages the needed blobs
-for these boards. This is because the upstream `linux-firmware` doesn't support
-or has outdated files for the blobs we need. The recipe
-`linux-firmware-raspbian` is based on a fork of `linux-firmware` which
-includes everything we need in order to fully support the WiFi hardware. All
-machines define `MACHINE_EXTRA_RRECOMMENDS` accordingly.
+## WiFi and Bluetooth Firmware
+
+Be aware that the WiFi and Bluetooth firmware for the supported boards
+is not available in stock `linux-firmware`.  These are added from
+Raspbian repositories.  All machines define `MACHINE_EXTRA_RRECOMMENDS`
```

Comment:
NIT: Extra space before `All`.
As well I would reword `stock` by referencing the upstream provider of that recipe, which is poky.

Reply:
I'm of the "Two spaces after period" generation.  I can change it, but can't promise it won't recur.

Reply:
Got it. I can't promise that I won't complain :) . Thanks.